src/world/Action/EffectBuilder.cpp

```diff
@@ -31,62 +31,229 @@ uint64_t EffectBuilder::getResultDelayMs()
   return Common::Util::getTimeMs() + 850;
 }
 
-EffectResultPtr EffectBuilder::getResult( Entity::CharaPtr& chara )
+std::shared_ptr< std::vector< EffectResultPtr > > EffectBuilder::getResultList( Entity::CharaPtr& chara )
```

Comment:
this doesn't really need to be a `shared_ptr`, should just use a `unique_ptr`

Reply:
or just return a reference to the vector, bit nicer

Reply:
or not returning anything at all :)
see the latest update

Reply:
lol, works for me, looks good